sysbuild/suit.cmake

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(SB_CONFIG_SUIT_BUILD_FLASH_COMPANION)
```

Comment:
But why is this placed here instead of https://github.com/nrfconnect/sdk-nrf/blob/main/cmake/sysbuild/suit.cmake